systemcontractindex/stakingindex/bucket.go

```diff
@@ -0,0 +1,124 @@
+package stakingindex
```

Comment:
these new codes are same as the existing staking contract indexer?
If that's the case, we should delete the existing code?

Reply:
existed staking indexer will be refacted by the new code in future